### PR TITLE
LL-1633: Add LiveQR export on mobile

### PR DESCRIPTION
**Pull request by pbondoer**
This PR adds the LiveQR export functionality to mobile to be able to import from another mobile device.

For now it's hidden in the :sparkles: secret debug menu :sparkles: but it will be more user-facing #soon

### :movie_camera: Video demo

[Click here](
https://www.youtube.com/watch?v=oUVsdabfl84)

https://www.youtube.com/watch?v=oUVsdabfl84

### :ok_hand: Type

Feature

### :mag: Context

LL-1633

### :clipboard: Parts of the app affected / Test plan

* Debug menu > Export accounts
* Import accounts

### CODE REVIEW

File: src/reducers/settings.js [hunk 1]
```diff
@@ -323,4 +323,15 @@ export const countervalueFirstSelector = (state: State) =>
...
   Platform.OS !== "android" && state.settings.readOnlyModeEnabled;
 
+// $FlowFixMe
+export const exportSettingsSelector = createSelector(
+  counterValueCurrencySelector,
+  state => state.settings.currenciesSettings,
+  (counterValueCurrency, currenciesSettings) => ({
+    counterValue: counterValueCurrency.ticker,
+    currenciesSettings,
+    developerModeEnabled: __DEV__,
```
Comment (gre) [inline src/reducers/settings.js]: mmh, it's more the actual value of the settings developerModeEnabled. (this enable or not the testnet coins)

Comment (gre) [reply src/reducers/settings.js]: also, the signature of that liveqr settings have recently changed https://github.com/LedgerHQ/ledger-live-common/blob/master/src/cross.js (in recent live-common version)
and it is already the case on desktop : https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/reducers/settings.js#L248-L259

depending on which version of live-common we are on, we might want to HODL this PR for this new live-common version and adapt the present code.

Comment (pbondoer) [reply src/reducers/settings.js]: Couldn't understand how `pairExchanges` was stored on mobile, and for dev mode I couldn't find the switch in the settings reducer either. I'll look again tomorow :)

Comment (pbondoer) [reply src/reducers/settings.js]: Couldn't understand how `pairExchanges` was stored on mobile, and for dev mode I couldn't find the switch in the settings reducer either. I'll look again tomorow :)

**Comment by pbondoer** [comment]
After internal discussion we'll wait for `topic/erc20` to merge :ok_hand:

**Comment by gre** [comment]
@pbondoer it's low in this current sprint but you should later be able to rebase develop and have latest stuff that was blocking this work :)

**Comment by pbondoer** [comment]
@gre @Arnaud97234 Since we merged `topic/erc20` I have pushed some changes, could you please review again? :mag:

File: src/reducers/settings.js [hunk 2]
```diff
@@ -85,6 +86,7 @@ const INITIAL_STATE: SettingsState = {
   readOnlyModeEnabled: !Config.DISABLE_READ_ONLY,
   experimentalUSBEnabled: false,
   countervalueFirst: false,
+  developerMode: !!__DEV__,
```
Comment (gre) [inline src/reducers/settings.js]: mmh I don't think this is what we want. the developerModeEnabled is actually mapped to `getEnv("DEV_MODE")`, which is **not** the same as __DEV__. it's basically a user's settings that enable or not the testnet apps and currencies.

Comment (gre) [reply src/reducers/settings.js]: (it's ok to call getEnv("DEV_MODE") at runtime from the selector because it's going to be set when you enter the screen)

Comment (pbondoer) [reply src/reducers/settings.js]: @gre This is what we do on desktop though... https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/src/reducers/settings.js#L88

Comment (gre) [reply src/reducers/settings.js]: I think that `developerMode` is a dead code unfortunately! seems like it was supposed to get dropped as we move it all to the DEV_MODE env. it's used in bunch of place in live-common and is the way we filter apps and currencies.

Comment (gre) [reply src/reducers/settings.js]: the question is do we restore it back correctly, i think we don't. i wonder if we want to really use that in the "export" if it's not really working. We might not want to export the experimental settings. 

wdyt @aramab  ?

Comment (pbondoer) [reply src/reducers/settings.js]: Export is definitely using that, I followed the selector there, so if there's changes to make there I'll port them over to desktop too :)

Comment (aramab) [reply src/reducers/settings.js]: I would export it IMO

**Pull request merged**